src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -0,0 +1,94 @@
+<Project Sdk="Microsoft.NET.Sdk.IL">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimum);netstandard2.0;net7.0</TargetFrameworks>
```

Comment:
I wonder what would happen if someone references this library on the .NET 6 out-of-support runtime. Would it work or blow up epically?

Reply:
How should I test this?

Reply:
Test the package with a .NET 6 runtime?

Reply:
All NETCoreApp versions are marked as placeholders:
```xml
<IsPlaceholderTargetFramework Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">true</IsPlaceholderTargetFramework>
```
Let me see what happens.

Reply:
> All NETCoreApp versions are marked as placeholders:

I'm not sure what you mean by that. We only target one .NETCoreApp version in this project: net7.0.

Reply:
Ok I see, the placeholder does not apply here even though the condition is stated as I pasted above. This means that if we reference this library via .NET 6.0, we would go through .NET Standard 2.0: https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0

Reply:
Correct

Reply:
@ericstj I tested consuming this new package using net6.0 and posted the result here: https://github.com/dotnet/maintenance-packages/pull/59#issuecomment-2256761751 

Since net6.0 is not yet out of support, I am unsure what to expect. I suppose we will end up trying to consume the netstandard2.0 version of the package that comes out from here, but then what?